exotations/task_maps/task_map/src/Point2Line.cpp

```diff
@@ -91,9 +91,18 @@ void Point2Line::update(Eigen::VectorXdRefConst x, Eigen::VectorXdRef phi, Eigen
         // direction from point to line
         const Eigen::Vector3d dv = direction(p);
         phi.segment<3>(i * 3) = dv;
-        for (int j = 0; j < J.cols(); j++)
+
+        if ((dv + p - line_start).norm() < 1e-9)
```

Comment:
How did you pick ``1e-9``? And could you use the system defined precision limits here?

Reply:
I thought there is no default epsilon in EXOTica? (I asked about this a while ago.)